base/src/org/compiere/model/MLandedCostAllocation.java

```diff
@@ -274,4 +275,63 @@ public boolean isReversalParent() {
 		return false;
 	}
 	
+	/**
+	 * getAmt 
+	 * Get Amount without Adjusted Credit Memo
+	 */
+	@Override
+	public BigDecimal getAmt() {
+		return getAmt(false, false);
+	}
+	
+	/**
+	 * Get Amount with Adjusted Credit Memo parameter
+	 * @param creditMemoAdjusted
+	 * @return
+	 */
+	public BigDecimal getAmt(boolean creditMemoAdjusted, boolean converted) {

```

Comment:
Please not create method only to adding complex logic , you can implement this logic into the getPriceActualCurrency() and not extent other method that adding more completely the code